dist/index.html

```diff
@@ -24,37 +24,62 @@
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                </div>
-            </div>
-        </nav>
-        <div class="headline">
-            <h2>"Hello and Welcome!"</h2>
-            <h1>TECHEX </br> Blockchain-Expo 2023</h1>
-        </div>
-        <div class="headline-box">
-            <p>
-                This two day event will take place in November 2023, featuring 20
-                speakers and more than 500 participants. The theme for the two-day
-                Block-Chain Expo: Connecting the Blockchain Ecosystem
-            </p>
+    <div class="desktop-banner">
+        <div class="desktop-nav-banner">
+            <a href="#"><i class='fa fa-facebook'></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
         </div>
-        <div class="headline-date">
-            <h3>30 November ~ 1 December 2023</h3>
-            <p>@ Crystal Blue Tech Conference Hall</p>
+        <div class="desktop-nav">
+            <div class="banner-logo">
+                <a href="./index.html"><img src="./assets/logo/TechEx-Main-White-Small.png" alt="techexogo"></a>
+            </div>
+            <div>
+                <ul class="desktop-nav-list">
+                    <li> <a href="./about.html">About</a> </li>
+                    <li> <a href="#">Program</a> </li>
+                    <li> <a href="#">Join</a> </li>
+                    <li> <a href="#">Sponsor</a> </li>
+                    <li> <a href="#">News</a> </li>
+                </ul>
+            </div>
+            <div class="nav-campain">
+                <p>Our&nbsp;Campain</p>
+            </div>
         </div>
     </div>
+    <nav class="nav-wrapper">
+        <div class="mobile-nav">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </div>
+    </nav>
 </header>
-                            <!-- Header End-->
-
 <main>
-            <!--     Programs Start   -->
+    <section class="headline-section">
+        <div class="header-container">
+            <div class="headline">
+                <h2>"Hello and Welcome!"</h2>
+                <h1>TECH-EX </br> Blockchain-Expo 2023</h1>
+            </div>
+            <div class="headline-box">
+                <p>
+                    This two day event will take place in November 2023, featuring 20
+                    speakers and more than 500 participants. The theme for the two-day
+                    Block-Chain Expo: Connecting the Blockchain Ecosystem
+                </p>
+            </div>
+            <div class="headline-date">
+                <h3>30 November ~ 1 December 2023</h3>
+                <p>@ Crystal Blue Tech Conference Hall</p>
+            </div>
+        </div>
+    </section>
```

Comment:
- Hey, let's kindly improve our design on this headline section to match that on the main design on the desktop version
   
   Improve the following:
   - font size
   - spacing
   - font-weight
   - align the contents correctly
> Try to improve the section to be close to the original design. :pray: 


*main design*
![image](https://github.com/mohashyne/Module1_Capstone/assets/59397159/5046707b-3abc-4d21-a1e8-bed8ff98fe46)

*your design*
![image](https://github.com/mohashyne/Module1_Capstone/assets/59397159/4188e9a1-7dd4-48d6-b613-8d20bf6093db)

Reply:
Request  changes fixed 😎